src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublishReleaseDataFunction.cs

```diff
@@ -164,5 +154,14 @@ public DataFactoryClientConfiguration(IConfiguration configuration)
                 TenantId = configuration.GetValue<string>(nameof(TenantId));
             }
         }
+        private bool ReleaseHasSubjects(Guid releaseId)
```

Comment:
Should have a space between functions

Reply:
Fixed. VS Code has this "1 Reference" line appear above functions that makes it look like there is a space when there isn't.